### DISCUSSION TITLE
Branches

### DISCUSSION THREAD

**Discussion by smichel17** [PaperWM-community/PaperWM#4] [General]
I opened https://github.com/paperwm/PaperWM/pull/396 from `next-release` here to `next-release` upstream.

My idea was to leave the develop branch with the notice as the default branch here (so people who randomly arrive here will see it), but continue working on `next-release`, where the PR is from.

@michaelhthomas Do you mind if I change the default branch back, revert https://github.com/PaperWM-community/PaperWM/commit/3e4dcd1f4506670f626cd2176979ded2757235f0 (so it isn't in the PR) and add the explanation above to the README on `develop`, telling people to look at `next-release`?

**Comment by michaelhthomas** [comment]
No definitely not, that makes much more sense from a PR perspective. As long as there's a notice that active development is on the next-release branch, I'm cool with it.

**Comment by smichel17** [comment]
I added a note to the `develop` branch readme, and changed it back to the default branch — So @Gelbana @tadfisher @paradajz you'll need to change the branch when opening new PRs.
